firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -282,86 +289,4 @@ class SearchTests: BaseTestCase {
         waitForTabsButton()
         validateSearchSuggestionText(typeText: "localhost")
     }
-// TODO: Add UI Tests back when felt privay simplified UI feature flag is enabled or when we support feature flags for tests
```

Comment:
Not sure about removing these tests... they were added very recently by an iOS developer in this commit: https://github.com/mozilla-mobile/firefox-ios/commit/f2edd0beecb126bb7a0e38541619571fb3062d30

The feature is there and could be tested manually but not in automation because it is behind a feature flag. 

Could you check with @abodea if we have similar tests for this? or what's the status?

Reply:
Added the tests back until we have a clear view for the next steps.